packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/README.md

```diff
@@ -18,8 +18,9 @@
 14. [Dev Menu](#dev-menu){{/enable_dev_menu}}{{#enable_patrol}}
 15. [Patrol integration tests](#patrol-integration-tests){{/enable_patrol}}{{#realtime_communication}}
 16. [Realtime communication](#realtime-communication){{/realtime_communication}}{{#enable_feature_otp}}
-17. [Feature OTP](#feature-otp){{/enable_feature_otp}}
-18. [Next Steps](#next-steps)
+17. [Feature OTP](#feature-otp){{/enable_feature_otp}} {{#cicd}}
+18. [CI/CD](#cicd){{/cicd}}
```

Comment:
This doesn't get generated when I try creating the project with fastlane enabled

Reply:
Are you sure? I tried running it in all three scenarios: 
1. `--cicd=fastlane`
2. `--cicd=none`
3.  without the `--cicd=(...)` parameter

The first and third approach properly generated the contents of cicd fastlane feature, while the second didn't implement the feature as part of the generated project/

Reply:
I tried generating it using `--interactive`